test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -116,7 +116,6 @@ const (
 
 	// singleCallTimeout is how long to try single API calls (like 'get' or 'list'). Used to prevent
 	// transient failures from failing tests.
-	// TODO: client should not apply this timeout to Watch calls. Increased from 30s until that is fixed.
 	singleCallTimeout = 5 * time.Minute
```

Comment:
can we consider framework.SingleCallTimeout about this timeout?

Reply:
The name(singleCallTimeout) and value(5 * time.Minute) are the same.
So it is nice to use framework.SingleCallTimeout instead, I feel.

Reply:
thanks. i think so.
Let's see what others opinion?
/cc @neolit123 
/cc @alejandrox1

Reply:
+1